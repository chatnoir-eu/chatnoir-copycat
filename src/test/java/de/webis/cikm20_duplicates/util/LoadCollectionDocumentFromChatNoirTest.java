package de.webis.cikm20_duplicates.util;

import org.approvaltests.Approvals;
import org.junit.Test;

import net.jodah.failsafe.RetryPolicy;

public class LoadCollectionDocumentFromChatNoirTest {

	RetryPolicy<String> NO_RETRIES = new RetryPolicy<String>()
			.handle(Exception.class)
			.withMaxRetries(1);
	
	@Test(expected = Exception.class)
	public void checkThatNonExistingCw09IdThrowsException() {
		String documentId = "clueweb09-en0004-47-2719612212";
		CollectionDocumentUtil.loadCollectionDocument(documentId, NO_RETRIES);
	}
	
	@Test(expected = Exception.class)
	public void checkThatNonExistingCw12IdThrowsException() {
		String documentId = "clueweb12-0900wb-15-0870012";
		CollectionDocumentUtil.loadCollectionDocument(documentId, NO_RETRIES);
	}
	
	@Test
	public void approveCrawlingOfExampleCw09Document() {
		String documentId = "clueweb09-en0004-47-27196";
		
		Approvals.verify(CollectionDocumentUtil.loadCollectionDocument(documentId));
	}
	
	@Test
	public void approveCrawlingOfExampleCw12Document() {
		String documentId = "clueweb12-0900wb-15-08700";
		
		Approvals.verify(CollectionDocumentUtil.loadCollectionDocument(documentId));
	}
}
