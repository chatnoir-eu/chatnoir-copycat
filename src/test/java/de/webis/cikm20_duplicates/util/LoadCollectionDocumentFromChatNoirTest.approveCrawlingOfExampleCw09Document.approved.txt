{"id":"clueweb09-en0004-47-27196","content":"Pennsylvania PA Hotels Motels B&B | AllStays Guide  Sitemap: Pennsylvania Hotels | US Hotels Motels & Lodging | Also under inns lodges   Members | Support   Hotel City Search: Search  Travel News  Travel Talk  USA  Canada  Mexico  Caribbean  Europe  Central & South America  Africa  Asia  Australia  Help  Luxury Hotels   |   Spa Resorts   |   Eco & Green   |   Bed & Breakfasts   |   Ski Resorts   |   Hostels   |   Haunted Hotels   |   Campgrounds    Extended Stays   |   Yoga Retreats   |   Real Suites   |   Family Friendly   |   Pet-Friendly   |   By Chain   |   Lighthouses   |        RV Parks   Why AllStays? We're for travelers, not just commissions. No one else links to official websites and provides direct phone numbers.     Stay anywhere...     PENNSYLVANIA HOTELS & LODGING     Email this |  del.icio.us |  Google Bookmark |  Live |  Facebook |  Yahoo |  Stumble |  Map  Check-in: Month January February March April May June July August September October November December 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 2009 2010 Check-out: Month January February March April May June July August September October November December 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 2009 2010  Rooms:  1 2 3 4 Adults:  1 2 3 4  Children:  0 1 2 3 4 (per room) Our Pennsylvania Guides Pennsylvania Hotels Pennsylvania Hotels on Maps Pennsylvania Bed & Breakfasts Pennsylvania Campgrounds Pennsylvania Convention Hotels Philadelphia Cruise Port Hotels Pennsylvania Extended Stays Pennsylvania Green Hotels Pennsylvania Haunted Hotels Pennsylvania Luxury Hotels Pennsylvania RV Parks Pennsylvania Ski Hotels Pennsylvania Spa Hotels Pennsylvania Suites Hotels Pennsylvania Interstate Exits Pennsylvania Internet Camping Pennsylvania State Parks Camping Pennsylvania State Parks by Map Pennsylvania Data & Statistics Pennsylvania Weather U.S. Embassy Information Free Pennsylvania Travel Planner               We hope you like our unique guide. Please visit again. Even if you book your trip elsewhere, use us for research, official websites and phone numbers. We want to be your world travel website starting point. All Available Discounts (Age, Clubs, Groups, Rewards Programs)  Go direct. Put your reservation at the front desk. Use these text guides for all hotels, official websites & phone numbers. We are the only hotel website to connect you with the real source. Select Pennsylvania Town... Aaronsburg Hotels Adamstown Hotels Akron Hotels Allensville Hotels Allentown Hotels Altoona Hotels Avella Hotels Barkeyville Hotels Bartonsville Hotels Beaver Falls Hotels Bedford Hotels Belle Vernon Hotels Bellefonte Hotels Bensalem Hotels Bentleyville Hotels Berwyn Hotels Bethel Hotels Bethlehem Hotels Bird in Hand Hotels Birdsboro Hotels Blairsville Hotels Blakeslee Hotels Bloomsburg Hotels Blue Ball Hotels Boalsburg Hotels Boiling Springs Hotels Bradford Hotels Breezewood Hotels Breinigsville Hotels Bridgeville Hotels Bristol Hotels Brodheadsville Hotels Brookville Hotels Burnham Hotels Bushkill Hotels Butler Hotels Camp Hill Hotels Canadensis Hotels Canonsburg Hotels Carlisle Hotels Carnegie Hotels Cashtown Hotels Chadds Ford Hotels Chalk Hill Hotels Chambersburg Hotels Chester Heights Hotels Churchtown Hotels Clarion Hotels Clarks Summit Hotels Clearfield Hotels Collegeville Hotels Columbia Hotels Concordville Hotels Conshohocken Hotels Coraopolis Hotels Corry Hotels Cranberry Township Hotels Crystal Spring Hotels Danville Hotels Delaware Water Gap Hotels Delmont Hotels Denver Hotels Dickson City Hotels Dillsburg Hotels Donegal Hotels Douglassville Hotels Doylestown Hotels Drums Hotels DuBois Hotels Duncansville Hotels Dunmore Hotels Eagles Mere Hotels East Berlin Hotels East Earl Hotels East Stroudsburg Hotels Easton Hotels Ebensburg Hotels Elizabethtown Hotels Elysburg Hotels Enola Hotels Ephrata Hotels Erie Hotels Erwinna Hotels Essington Hotels Etters Hotels Equinunk Hotels Eufala Hotels Exton Hotels Fairfield Hotels Fairview Hotels Farmington Hotels Fawn Grove Hotels Fayette City Hotels Fayetteville Hotels Fort Washington Hotels Frackville Hotels Franklin Hotels Frazer Hotels Galeton Hotels Gardenville Hotels Gardners Hotels Gettysburg Hotels Gibsonia Hotels Glen Mills Hotels Gordonville Hotels Grantville Hotels Greencastle Hotels Greensburg Hotels Grove City Hotels Hamlin Hotels Hanover Hotels Harmarville Hotels Harrisburg Hotels Hatfield Hotels Hawley Hotels Hazleton Hotels Hermitage Hotels Hershey Hotels Hilltown Hotels Holicong Hotels Honesdale Hotels Horsham Hotels Hummelstown Hotels Huntingdon Hotels Indiana Hotels Intercourse Hotels Jacobus Hotels Jim Thorpe Hotels Johnstown Hotels Jonestown Hotels Kennett Square Hotels King of Prussia Hotels Kintnersville Hotels Kittanning Hotels Kulpsville Hotels Kutztown Hotels Lackawaxen Hotels Lafayette Hill Hotels Lake Harmony Hotels Lakeville Hotels Lampeter Hotels Lancaster Hotels Landenberg Hotels Langhorne Hotels Lebanon Hotels Leisenring Hotels Lester Hotels Levittown Hotels Lewisburg Hotels Ligonier Hotels Lock Haven Hotels Malvern Hotels Manheim Hotels Mansfield Hotels Marietta Hotels Markleton Hotels Mars Hotels Marshalls Creek Hotels Matamoras Hotels Maytown Hotels Meadville Hotels Mechanicsburg Hotels Media Hotels Mendenhall Hotels Mercer Hotels Mercersburg Hotels Middlebury Center Hotels Middletown Hotels Mifflintown Hotels Mifflinville Hotels Milesburg Hotels Milford Hotels Mill Hall Hotels Mill Run Hotels Millersville Hotels Milroy Hotels Monaca Hotels Monroeville Hotels Montgomeryville Hotels Montoursville Hotels Moosic Hotels Morgantown Hotels Morrisville Hotels Mount Gretna Hotels Mount Joy Hotels Mount Pleasant Hotels Mount Pocono Hotels Mountville Hotels Muncy Hotels New Berlin Hotels New Castle Hotels New Columbia Hotels New Cumberland Hotels New Holland Hotels New Hope Hotels New Kensington Hotels New Kingstown Hotels New Milford Hotels New Oxford Hotels New Stanton Hotels New Tripoli Hotels North East Hotels North Huntingdon Hotels North Wales Hotels Oakdale Hotels Oaks Hotels Ohiopyle Hotels Oil City Hotels Oxford Hotels Paradise Hotels Philadelphia Hotels Pine Grove Hotels Pittsburgh Hotels Pittston Hotels Pleasant Hills Hotels Plymouth Meeting Hotels Pocono Manor Hotels Point Pleasant Hotels Port Matilda Hotels Pottstown Hotels Pottsville Hotels Punxsutawney Hotels Quakertown Hotels Quarryville Hotels Railroad Hotels Reading Hotels Red Lion Hotels Rockwood Hotels Ronks Hotels Rushland Hotels Saint Davids Hotels Saint Marys Hotels Sayre Hotels Schellsburg Hotels Scotrun Hotels Scranton Hotels Selinsgrove Hotels Shamokin Dam Hotels Sharon Hotels Shawnee On Delaware Hotels Shinglehouse Hotels Shippensburg Hotels Skytop Hotels Smoketown Hotels Somerset Hotels Stahlstown Hotels State College Hotels Sterling Hotels Strasburg Hotels Stroudsburg Hotels Tannersville Hotels Tarentum Hotels Terre Hill Hotels Titusville Hotels Towanda Hotels Trevose Hotels Troy Hotels Tunkhannock Hotels Uniontown Hotels Upper Black Eddy Hotels Valley Forge Hotels Warfordsburg Hotels Warminster Hotels Warren Hotels Washington Hotels Washington Crossing Hotels Wattsburg Hotels Wayne Hotels Waynesboro Hotels Waynesburg Hotels Wellsboro Hotels West Chester Hotels West Conshohocken Hotels West Hazleton Hotels West Homestead Hotels West Middlesex Hotels West Mifflin Hotels West Newton Hotels White Haven Hotels Wilkes-Barre Hotels Williamsport Hotels Willow Grove Hotels Willow Street Hotels Winfield Hotels Wormleysburg Hotels Wyomissing Hotels Wysox Hotels Yardley Hotels York Hotels HELP How to Use AllStays Book online: Enter your city name and dates for online booking only. Book direct: Use the middle column drop down box for complete hotel, motel, and B&B listings for all across Pennsylvania. We hope you reserve with our own booking links but you can call or click on the property name for official website when it exists, and book direct. Customer Support Retrieve or Cancel Reservations Other websites only list hotels that pay them a commission, with the highest profits at the top of their results. And add on booking fees. We give you all the choices. Period. Read reviews. View maps. Make your choice. About | Site Map | Privacy Policy | Terms | Add Your Property | Support | Book online with us or go direct ©1999-2009 AllStays. All rights reserved. Sitemap: Pennsylvania Hotels & Motels Location Guide","fullyCanonicalizedContent":"pennsylvania pa hotel motel b b allstai guid sitemap pennsylvania hotel u hotel motel lodg also under inn lodg member support hotel citi search search travel new travel talk usa canada mexico caribbean europ central south america africa asia australia help luxuri hotel spa resort eco green bed breakfast ski resort hostel haunt hotel campground extend stai yoga retreat real suit famili friendli pet friendli chain lighthous rv park why allstai we'r travel just commiss on els link offici websit provid direct phone number stai anywher pennsylvania hotel lodg email del.icio.u googl bookmark live facebook yahoo stumbl map check month januari februari march april mai june juli august septemb octob novemb decemb 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 2009 2010 check out month januari februari march april mai june juli august septemb octob novemb decemb 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 2009 2010 room 1 2 3 4 adult 1 2 3 4 children 0 1 2 3 4 per room our pennsylvania guid pennsylvania hotel pennsylvania hotel map pennsylvania bed breakfast pennsylvania campground pennsylvania convent hotel philadelphia cruis port hotel pennsylvania extend stai pennsylvania green hotel pennsylvania haunt hotel pennsylvania luxuri hotel pennsylvania rv park pennsylvania ski hotel pennsylvania spa hotel pennsylvania suit hotel pennsylvania interst exit pennsylvania internet camp pennsylvania state park camp pennsylvania state park map pennsylvania data statist pennsylvania weather u. embassi inform free pennsylvania travel planner we hope you like our uniqu guid pleas visit again even you book your trip elsewher us u research offici websit phone number we want your world travel websit start point all avail discount ag club group reward program go direct put your reserv front desk us text guid all hotel offici websit phone number we onli hotel websit connect you real sourc select pennsylvania town aaronsburg hotel adamstown hotel akron hotel allensvil hotel allentown hotel altoona hotel avella hotel barkeyvil hotel bartonsvil hotel beaver fall hotel bedford hotel bell vernon hotel bellefont hotel bensalem hotel bentleyvil hotel berwyn hotel bethel hotel bethlehem hotel bird hand hotel birdsboro hotel blairsvil hotel blakesle hotel bloomsburg hotel blue ball hotel boalsburg hotel boil spring hotel bradford hotel breezewood hotel breinigsvil hotel bridgevil hotel bristol hotel brodheadsvil hotel brookvil hotel burnham hotel bushkil hotel butler hotel camp hill hotel canadensi hotel canonsburg hotel carlisl hotel carnegi hotel cashtown hotel chadd ford hotel chalk hill hotel chambersburg hotel chester height hotel churchtown hotel clarion hotel clark summit hotel clearfield hotel collegevil hotel columbia hotel concordvil hotel conshohocken hotel coraopoli hotel corri hotel cranberri township hotel crystal spring hotel danvil hotel delawar water gap hotel delmont hotel denver hotel dickson citi hotel dillsburg hotel doneg hotel douglassvil hotel doylestown hotel drum hotel duboi hotel duncansvil hotel dunmor hotel eagl mere hotel east berlin hotel east earl hotel east stroudsburg hotel easton hotel ebensburg hotel elizabethtown hotel elysburg hotel enola hotel ephrata hotel eri hotel erwinna hotel essington hotel etter hotel equinunk hotel eufala hotel exton hotel fairfield hotel fairview hotel farmington hotel fawn grove hotel fayett citi hotel fayettevil hotel fort washington hotel frackvil hotel franklin hotel frazer hotel galeton hotel gardenvil hotel gardner hotel gettysburg hotel gibsonia hotel glen mill hotel gordonvil hotel grantvil hotel greencastl hotel greensburg hotel grove citi hotel hamlin hotel hanov hotel harmarvil hotel harrisburg hotel hatfield hotel hawlei hotel hazleton hotel hermitag hotel hershei hotel hilltown hotel holicong hotel honesdal hotel horsham hotel hummelstown hotel huntingdon hotel indiana hotel intercours hotel jacobu hotel jim thorp hotel johnstown hotel jonestown hotel kennett squar hotel king prussia hotel kintnersvil hotel kittan hotel kulpsvil hotel kutztown hotel lackawaxen hotel lafayett hill hotel lake harmoni hotel lakevil hotel lampet hotel lancast hotel landenberg hotel langhorn hotel lebanon hotel leisenr hotel lester hotel levittown hotel lewisburg hotel ligoni hotel lock haven hotel malvern hotel manheim hotel mansfield hotel marietta hotel markleton hotel mar hotel marshal creek hotel matamora hotel maytown hotel meadvil hotel mechanicsburg hotel media hotel mendenhal hotel mercer hotel mercersburg hotel middleburi center hotel middletown hotel mifflintown hotel mifflinvil hotel milesburg hotel milford hotel mill hall hotel mill run hotel millersvil hotel milroi hotel monaca hotel monroevil hotel montgomeryvil hotel montoursvil hotel moosic hotel morgantown hotel morrisvil hotel mount gretna hotel mount joi hotel mount pleasant hotel mount pocono hotel mountvil hotel munci hotel new berlin hotel new castl hotel new columbia hotel new cumberland hotel new holland hotel new hope hotel new kensington hotel new kingstown hotel new milford hotel new oxford hotel new stanton hotel new tripoli hotel north east hotel north huntingdon hotel north wale hotel oakdal hotel oak hotel ohiopyl hotel oil citi hotel oxford hotel paradis hotel philadelphia hotel pine grove hotel pittsburgh hotel pittston hotel pleasant hill hotel plymouth meet hotel pocono manor hotel point pleasant hotel port matilda hotel pottstown hotel pottsvil hotel punxsutawnei hotel quakertown hotel quarryvil hotel railroad hotel read hotel red lion hotel rockwood hotel ronk hotel rushland hotel saint david hotel saint mari hotel sayr hotel schellsburg hotel scotrun hotel scranton hotel selinsgrov hotel shamokin dam hotel sharon hotel shawne delawar hotel shinglehous hotel shippensburg hotel skytop hotel smoketown hotel somerset hotel stahlstown hotel state colleg hotel sterl hotel strasburg hotel stroudsburg hotel tannersvil hotel tarentum hotel terr hill hotel titusvil hotel towanda hotel trevos hotel troi hotel tunkhannock hotel uniontown hotel upper black eddi hotel vallei forg hotel warfordsburg hotel warminst hotel warren hotel washington hotel washington cross hotel wattsburg hotel wayn hotel waynesboro hotel waynesburg hotel wellsboro hotel west chester hotel west conshohocken hotel west hazleton hotel west homestead hotel west middlesex hotel west mifflin hotel west newton hotel white haven hotel wilk barr hotel williamsport hotel willow grove hotel willow street hotel winfield hotel wormleysburg hotel wyomiss hotel wysox hotel yardlei hotel york hotel help how us allstai book onlin enter your citi name date onlin book onli book direct us middl column drop down box complet hotel motel b b list all across pennsylvania we hope you reserv our own book link you can call click properti name offici websit when exist book direct custom support retriev cancel reserv other websit onli list hotel pai them commiss highest profit top result add book fee we give you all choic period read review view map make your choic about site map privaci polici term add your properti support book onlin u go direct 1999 2009 allstai all right reserv sitemap pennsylvania hotel motel locat guid","url":null}